data/plans/global.json

```diff
@@ -12,7 +12,7 @@
     "support_level": "Basic",
     "support_list": [
       "Help centre",
-      "Community forum support"
+      "Community forum"
```

Comment:
Don't we want to remove it? We're removing it in the Investapp?

Reply:
Good point - the requirement says :

> Remove "support" copy next to "community forum" on pricing page

I'll check.

Reply:
Can confirm we're keeping this copy 🤷‍♂️

Reply:
I guess then I'd push back onto the Product one…?  They should be the same 😕…oh well, once we rebuild these pages, they'll be the same or more-so likely entirely discarded.